mqtt-streaming/src/test/scala/docs/scaladsl/MqttSessionSpec.scala

```diff
@@ -54,7 +54,7 @@ class MqttSessionSpec
               .clientSessionFlow(session)
               .join(pipeToServer)
           )
-          .toMat(Sink.collection)(Keep.both)
+          .toMat(Sink.seq)(Keep.both)
```

Comment:
Why this change?

Reply:
I see this issue remains also: https://github.com/akka/akka/issues/26305

Reply:
Yes, that is the reason. I think this solution is the best for now.